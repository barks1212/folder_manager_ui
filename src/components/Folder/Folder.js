import React from 'react';
import PropTypes from 'prop-types';

const folder = () => (
    <div>
        this is a folder
    </div>
)

export default folder;