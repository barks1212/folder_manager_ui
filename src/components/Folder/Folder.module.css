.Folder {
    background-color: red;
    width: 10rem;
    height: 5rem;
    border: 2px solid black;
    display: flex;
    justify-content: center;
    margin: 5px;
}

.Folder ul {
    list-style-type: none;
    margin: 0;
    padding: 20px;
    text-align: center;
}

.Name {
    font-weight: bold;
}

.Chev {
    color: white;
}
