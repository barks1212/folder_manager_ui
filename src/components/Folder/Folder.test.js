import React from 'react';
import { render, cleanup } from '@testing-library/react'

import Folder from './Folder';

afterEach(cleanup);

describe('folder', () => {
    it('should render', () => {
        const {asFragment} = render(<Folder />)
        expect(asFragment()).toMatchSnapshot();
    })
})