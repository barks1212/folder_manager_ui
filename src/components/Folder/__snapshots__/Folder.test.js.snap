// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`folder should render 1`] = `
<DocumentFragment>
  <div>
    this is a folder
  </div>
</DocumentFragment>
`;
